wagtail/admin/widgets/button.py

```diff
@@ -8,14 +8,21 @@
 
 class Button(Component):
     show = True
+    label = ""
+    icon_name = None
 
     def __init__(
-        self, label, url, classname="", icon_name=None, attrs={}, priority=1000
+        self, label="", url=None, classname="", icon_name=None, attrs={}, priority=1000
     ):
-        self.label = label
+        if label:
```

Comment:
It would have to be `self.label = label or self.label` to allow the kwarg to take precedence, and I think that's enough of a mental speed-bump to justify sticking with the `if label:` version.